gobblin-api/src/main/java/gobblin/util/ClassAliasResolver.java

```diff
@@ -38,16 +39,17 @@
  *
  * <br>
  * <b>
- *   Note: For the moment this class will only look for classes with gobblin prefix, as scanning the entire classpath is
- *   very expensive.
+ *   Note: For the moment this class will only look for classes with gobblin/com.linkedin.gobblin prefix, as scanning
+ *   the entire classpath is very expensive.
  * </b>
  */
 @Slf4j
 public class ClassAliasResolver<T> {
 
-  // Scan all packages in the classpath with prefix gobblin when class is loaded.
+  // Scan all packages in the classpath with prefix gobblin, com.linkedin.gobblin when class is loaded.
   // Since scan is expensive we do it only once when class is loaded.
-  private static final Reflections REFLECTIONS = new Reflections("gobblin");
+  private static final Reflections REFLECTIONS = new Reflections(new ConfigurationBuilder().forPackages("gobblin",
```

Comment:
This should be configurable. Also, why are we scanning classes within com.linkedin.gobblin package, when the packages are gobblin.* in our code base?